tests/test_checkpointing.py

```diff
@@ -232,7 +232,34 @@ def __init__(self, requires_grad):
             )
             assert len(pretrain_best_checkpoints) == 1
 
-        def test_does_produce_results(self):
-            file_path = "~/repo/sample_run/jiant-demo/results.tsv"
-            file = open(file_path, "rb")
-            assert file
+    def test_get_best_checkpointing(self):
```

Comment:
Add a brief comment saying what this is doing. It's not immediately obvious that this is just testing the flag/path logic—this method name suggests that we might actually be loading stuff.